shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/BridgeIngressServiceImpl.java

```diff
@@ -30,14 +33,48 @@
 
     @Override
     public void createBridgeIngress(BridgeDTO bridgeDTO) {
-        final Namespace namespace = customerNamespaceProvider.fetchOrCreateCustomerNamespace(bridgeDTO.getCustomerId());
+        final Namespace namespace = customerNamespaceProvider.getOrCreateCustomerNamespace(bridgeDTO.getCustomerId());
```

Comment:
@ricardozanini Is that ok if we rename to `getOrCreateCustomerNamespace` for you?

Reply:
TBH I don't like to use `get` since this verb is usually associated with accessors. But I'm not entirely against this change.

Reply:
It's fine, I've reverted the changes :+1: